app/views/teachers/progress_reports/_subnav.html.erb

```diff
@@ -4,7 +4,7 @@
       <li><%= active_link_to 'Activity Summary', scorebook_teachers_classrooms_path, class: 'default'%></li>
       <li><%= active_link_to 'Activity Analysis', '/teachers/progress_reports/diagnostic_reports/#/activity_packs', class: 'default activity-analysis-tab'%></li>
       <li><%= active_link_to 'Diagnostics', '/teachers/progress_reports/diagnostic_reports/#/diagnostics', class: 'default diagnostic-tab'%></li>
-      <li><%= active_link_to raw('Activity Scores  <i class="fa fa-star"></i>'), '/teachers/progress_reports/activities_scores_by_classroom'%></li>
+      <li><a href='/teachers/progress_reports/activities_scores_by_classroom' class=<%=%w(/teachers/progress_reports/student_overview /teachers/progress_reports/activities_scores_by_classroom).include?(request.path) ? 'active' : nil%> >Activity Scores  <i class="fa fa-star"></i></a></li>
```

Comment:
```/student_overview /teachers/```

Does the space here not cause an issue?

Reply:
@ddmck the space is intentional. It is a word array of the various paths.